dpti/hti.py

```
@@ -0,0 +1,1580 @@
+    sparam={},
+    switch="one-step",
+    step="both",
+    if_meam=False,
+    meam_model=None,
+    custom_variables=None,
+    append=None,
+):
+    ret = ""
+    ret += "clear\n"
+    ret += "# --------------------- VARIABLES-------------------------\n"
+    ret += "variable        NSTEPS          equal %d\n" % nsteps
```

Ruff (0.8.2)

Use format specifiers instead of percent format

(UP031)